karma.conf.js

```diff
@@ -101,6 +104,7 @@ module.exports = function(config) {
 
     babelPreprocessor: {
       options: {
+        presets: transpile ? ['es2015'] : [],
```

Comment:
If this is needed for Karma then it is also needed for the runtime transpiler, as well as `use_require.js`.